test/src/main/java/org/apache/accumulo/test/MetaConstraintRetryIT.java

```diff
@@ -52,15 +51,14 @@ public void test() throws Exception {
           TablePermission.WRITE);
 
       ServerContext context = getServerContext();
-      Writer w = new Writer(context, MetadataTable.ID);
       KeyExtent extent = new KeyExtent(TableId.of("5"), null, null);
 
       Mutation m = new Mutation(extent.toMetaRow());
       // unknown columns should cause constraint violation
       m.put("badcolfam", "badcolqual", "3");
-      var e = assertThrows(RuntimeException.class,
-          () -> MetadataTableUtil.update(context, w, null, m, extent));
-      assertEquals(ConstraintViolationException.class, e.getCause().getClass());
+      var e = assertThrows(IllegalArgumentException.class,
+          () -> MetadataTableUtil.update(context, null, m, extent));
+      assertEquals(MutationsRejectedException.class, e.getCause().getClass());
```

Comment:
I think the idea here was to verify that the rejection happened because the constraint was violated. I think you need to inspect the MutationsRejectedException a bit deeper in order to get the same verification.

Reply:
changed in a513d75